D:\Users\Papa\Documents\GIT\calibre>calibre-debug -c "from calibre.rpdb import cli; cli()"
Connecting to remote debugger on port 4444...
Connected to remote process
Traceback (most recent call last):
  File "runpy.py", line 194, in _run_module_as_main
  File "runpy.py", line 87, in _run_code
  File "site.py", line 82, in <module>
  File "site.py", line 77, in main
  File "site.py", line 49, in run_entry_point
  File "calibre\debug.py", line 270, in main
  File "<string>", line 1, in <module>
  File "calibre\rpdb.py", line 112, in cli
ModuleNotFoundError: No module named 'readline'
